src/output.jl

```diff
@@ -37,34 +37,50 @@ end
 Output() = Output{Float32}()
 
 """
-    run_id, run_path = get_run_id_path(P::Parameters)
-
-Checks existing `run????` folders in output path to determine a 4-digit `run_id` number
-and creates a new folder `run????` with that `run_id`. Also returns the full path
-`run_path` of that folder. Returns `0,"no runpath"` in the case of no output."""
-function get_run_id_path(P::Parameters)
+    run_id = get_run_id(output, output_path)
 
-    @unpack output,output_path = P
+Checks existing `run-????` folders in output path to determine a 4-digit `run_id` number. 
+"""
+function get_run_id(output, output_path)
 
-    if output
+    if output 
         # pull list of existing run???? folders via readdir
-        pattern = r"run\d\d\d\d"                # run???? in regex
+        pattern = r"run-\d\d\d\d"                # run-???? in regex
         runlist = filter(x->startswith(x,pattern),readdir(output_path))
         runlist = filter(x->endswith(  x,pattern),runlist)
-        existing_runs = [parse(Int,id[4:end]) for id in runlist]
+        existing_runs = [parse(Int,id[5:end]) for id in runlist]
```

Comment:
I believe this one will ignore any folders that don't follow the `run-????` with `?` being digits pattern. I guess that is intended?

Reply:
Yes, that's intended there. That's only for the automatically chosen names